actions/nix/update-nix-sources/update-nix-sources.py

```diff
@@ -186,9 +186,11 @@ def gh_add_pr_reviwers(pr_id: str, users: List[str]) -> None:
 
 
 
-def niv(cmd:str) -> None:
-  subprocess.run(["niv", cmd], check=True)
-
+def niv(cmd:str, source: str = None) -> None:
+  if source is None:
+    subprocess.run(["niv", cmd], check=True)
+  else:
+    subprocess.run(["niv", cmd, source], check=True)
```

Comment:
Kan du prøve dette istedet?
```suggestion
def niv(*cmd:str) -> None:
    cmds = ["niv"] + list(cmd)
    subprocess.run(cmds, check=True)
```

Reply:
Ja, tenkte også på noe lignende, det der er vel mer generaliserbart også